test/unit/exception_handling/mailer_test.rb

```diff
@@ -3,7 +3,11 @@
 module ExceptionHandling
   class MailerTest < ActionMailer::TestCase
 
-    include ActionDispatch::Assertions::SelectorAssertions
+    if ::Rails.const_defined?(:Dom)
```

Comment:
Will this always be defined in Rails 4? Ie, is this needed because it's only sometimes defined in Rails 4, or is it needed because it's not defined in Rails 3? And if it's the latter, do we need this if statement, or can we just change the include statement, since this branch will only be run with Rails 4?